src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -74,6 +81,7 @@ def __init__(self, storage=None, auth_ctx_factory=None):
         env = get_env()
         self._management_resource_uri = env[ENDPOINT_URLS.MANAGEMENT]
         self._graph_resource_uri = env[ENDPOINT_URLS.ACTIVE_DIRECTORY_GRAPH_RESOURCE_ID]
+        self._key_vault_resource_uri = env[ENDPOINT_URLS.KEY_VAULT]
```

Comment:
i don't think we will need `_graph_resource_uri` and `_key_vault_resource_uri`, just remove

The rest change on Profile LGTM